core/src/main/scala/chisel3/VerificationStatement.scala

```diff
@@ -56,6 +84,10 @@ object assert {
 
   import VerificationStatement._
 
+  @deprecated(
+    "This method has been deprecated in favor of _applyMacroWithStringMessage. Please use the same.",
+    "Chisel 3.5"
+  )
```

Comment:
This is not a public API? can we deprecate it?

Reply:
It is deprecated, or are you saying deprecate and just say "do not use"?

Reply:
From my understanding, it like any other Macro transformed APIs, they are not supported being called by users(but still public) since this PR removed APIs which used this macro, I thought it can be safely removed(rather deprecated)

Reply:
So while they absolutely shouldn't, users could technically define their own methods that [macro] call these: https://scastie.scala-lang.org/MmzrWNwvSyasKAUJPOIzgw. In any case I think we should deprecate on minor version and remove in 3.6.

Reply:
That's quite reasonable, although I doubt if there exists any user doing so. Thanks for the explanation! Let's go through the deprecation flow ;p